templates/openstack/what-is-openstack.html

```diff
@@ -701,7 +701,7 @@ <h3 class="p-heading--4">
             <a
               href="/engage/from-vmware-to-open-source"
```

Comment:
The link seems to have changed too:
```suggestion
              href="/engage/what-is-openstack"
```